CMakeLists.txt

```diff
@@ -201,6 +201,22 @@ cmake_dependent_option(
   "NOT EXECUTORCH_BUILD_ARM_BAREMETAL" OFF
 )
 
+#
+# portable_ops_lib: build portable_ops library. Disable, if not required
+#
+cmake_dependent_option(
+  BUILD_EXECUTORCH_PORTABLE_OPS "Build portable_ops library." ON
+  "NOT EXECUTORCH_BUILD_CADENCE" OFF
```

Comment:
I don't think we need to tie these together right. I think in the build commands you have for Cadence you can just explicitly turn BUILD_EXECUTORCH_PORTABLE_OPS off? Same with EXECUTORCH_USE_DL.

Reply:
Done